ui/page/dcrdex/dcrdex_page.go

```diff
@@ -101,19 +111,14 @@ func (pg *DEXPage) OnNavigatedTo() {
 		}
 
 		pg.dexIsLoading = false
+		pg.showSplashPage = false
```

Comment:
Move these code to before you call `pg.Display`.

Reply:
These values are updated in a goroutine and will be seen the the other process handling Layout(). This means we might end up not having a page to display until the next Layout() call.